conf/osp.yaml.template

```
@@ -0,0 +1,15 @@
+OSP:
+  # Openstack to be added as a compute resource
+  # hostname: Openstack api url For eg: https://hostname:port/v3/auth/tokens
```

```suggestion
  # Openstack api url For eg: https://hostname:port/v3/auth/tokens
```
You can remove the first part of all of these descriptions since they match the field they're describing.

---

done